include/utils/ISoilUtils.h

```diff
@@ -0,0 +1,89 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     MASTODON                  */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+// This code was implemented in colloboration with Ozgun Numanoglu
+// (numanog2@illinois.edu) and Omar Baltaji (baltaji2@illinois.edu) from UIUC.
+
+/**
+ * This namespace contains the functions used for the calculations corresponding
+ * to the I-Soil material model in ComputeISoilStress.
+ **/
+
+#ifndef ISOILUTILS_H
+#define ISOILUTILS_H
+
+// MOOSE includes
+#include "GeneralVectorPostprocessor.h"
+
+// Forward declarations
+namespace ISoilUtils
+{
+/// Deconstructs the backbone curve into a set of elastic-perfectly-plastic
+/// curves, each comprising a Young's modulus and yield stress.
+void computeSoilLayerProperties(std::vector<std::vector<Real>> & backbone_stress,
+                                std::vector<std::vector<Real>> & backbone_strain,
+                                const std::vector<unsigned int> & _layer_ids,
```

Comment:
Please remove the leading underscore on the variables throughout these functions